### ISSUE TITLE
Data Point missing in Charts

### ISSUE THREAD

**Issue by spatenchb** [lightmaster/Meteobridge-Weather34-Template#54]
William,
I noticed this morning (July 2 ) that the July 1 Max/Min data is not showing in my July temperature chart. Seems to be fine in all other charts. I checked the Jul.csv and the data is there.

I also checked your site and the data seems to be missing in most of your July charts. 

In my case, the cron ran and the data file was written, but the monthlytemperature.php does not seem to be pulling it.

Thanks,

Chuck

**Comment by hansmestrum** [comment]
Yes I also just noticed it.

**Comment by lightmaster** [comment]
I figured it out, its an issue with the interval that that chart uses. Gonna check a couple things with all the charts and then push out the change.

**Comment by lightmaster** [comment]
Wow there are a lot of charts to edit, lol. Making a number of other changes to address issues like how when there's only one day on a line graph, the graph is invisible unless you hover over it. Also fixing some of the formatting variables so the single days will be centered on the graph, rather than on the y axis all the way to the left.

![screenshot](https://i.postimg.cc/Bn3RC7zm/Screenshot-20190702-202409-Chrome-Beta.jpg)

**Comment by spatenchb** [comment]
Glad you found it! Yes, those charts did seem to multiply over the years.

**Comment by lightmaster** [comment]
It's funny (and annoying, lol) that it's several hours of work to make the dozen or so changes to all the charts, and that one simple image there sums up all of the changes.

Tied up with work for the next bit, but I've got about half the charts fixed. Regardless, need to wait until after midnight to make sure the ternary I added for the markers makes them go away when 2 or more data points exist. Don't want a line chart with a bunch of markers on it, especially with the yearly ones and their hundreds of points, lol.
